client/js/table-model.js

```diff
@@ -1,21 +1,21 @@
 class TableModel {
-	constructor(numCols=10, numRows=20) {
-		this.numCols = numCols;
-		this.numRows = numRows;
-		this.data = {};
-	}
+  constructor(numCols=10, numRows=20) {
```

Comment:
Please include a space before and after each operator, since that makes your code more human-readable.